### DISCUSSION TITLE
Downloading APK file. Authorization needed. Help please.

### DISCUSSION THREAD

**Discussion by Jiffcakez93** [Chokage/Fear-And-Hunger-Android-Application#3] [Q&A]
Hello Chokage, I came across your vid on youtube which brought me here. I am facing an issue with downloading The apk file as it keeps showing that i need authorization to download, mid download mind you. Is there any way you can help?

**Comment by Chokage** [comment]
That's quite odd. Normally there's no need for authorization for any of
this. There might be an issue with your browser's access to your device
files.

I'll take a look either way.

Meanwhile, try and download from this link:
https://github.com/Chokage/Fear-And-Hunger-Android-Application/releases/download/v1.0.3/Fear.and.Hunger.apk
